Helper/Invoice.php

```diff
@@ -258,7 +258,10 @@ public function createAdyenInvoice(
         $adyenInvoice = $this->adyenInvoiceFactory->create();
         $adyenInvoice->setPspreference($pspReference);
         $adyenInvoice->setAdyenPaymentOrderId($adyenOrderPayment[OrderPaymentInterface::ENTITY_ID]);
-        $adyenInvoice->setAmount($this->adyenDataHelper->originalAmount($captureAmountCents, $order->getBaseCurrencyCode()));
+        $adyenInvoice->setAmount($this->adyenDataHelper->originalAmount(
```

Comment:
Would it be safer to use the ChargedCurrency helper here? https://github.com/Adyen/adyen-magento2/blob/main/Helper/ChargedCurrency.php#L279-L293

Reply:
@acampos1916 you are right about this one, but we have an investigation ticket open for the bulk solution, we plan to go through all the fields that contain amount and currency and review whether we are including the charged currency logic to differentiate between the base amount/order amount and fix the places where we aren't